functionary/core/models/function.py

```diff
@@ -80,7 +81,18 @@ def deactivate(self):
         for scheduled_task in self.scheduled_tasks.all():
             scheduled_task.pause()
 
+    @property
+    def parameters(self):
+        """Convenience alias for functionparameter_set"""
+        # Provides better static type checking than using related_name
```

Comment:
I started doing this here and will probably go through and do it for everything at some point.  Basically, when we use `related_name`, django sets that up dynamically at runtime, which means the static type checker doesn't know about it.  Explicitly defining it as a property method fixes that for the type checking and also makes it more obvious to someone looking at this model that the relationship is there.

Reply:
No, not really unfortunately.  It returns a RelatedManager which is dynamically generated based on the manager of the model class.